tests/_bootstrap.php

```diff
@@ -1,5 +1,5 @@
 <?php
 // This is global bootstrap for autoloading 
-
-\Codeception\Util\Autoload::registerSuffix('Page', __DIR__.DIRECTORY_SEPARATOR.'_pages');
+\Codeception\Util\Autoload::registerSuffix('Page', __DIR__.DIRECTORY_SEPARATOR.'_pages/Joomla2/Administrator');
+\Codeception\Util\Autoload::registerSuffix('Page', __DIR__.DIRECTORY_SEPARATOR.'_pages/Joomla3/Administrator');
```

Comment:
I think that the Joomla Installation pages should be placed at /Joomla2/installation/ instead of Joomla2/adminstrator

What do you think?

![screen shot 2015-02-05 at 12 45 38](https://cloud.githubusercontent.com/assets/1375475/6059647/7403abec-ad34-11e4-89cb-33b0eba81ac9.png)